test/integration/fixtures/blocks/core__separator.html

```diff
@@ -1,3 +1,3 @@
-<!-- wp:core/separator -->
-<hr class="wp-block-separator" />
-<!-- /wp:core/separator -->
+<!-- wp:separator {"opacity":"alpha-channel"} -->
```

Comment:
Now that the `opacity` attribute is set to be `alpha-channel` by default, we can probably remove the explicit setting of the attribute in this fixture? When I add a block in the editor manually, its markup is now:

```
<!-- wp:separator -->
<hr class="wp-block-separator has-alpha-channel-opacity"/>
<!-- /wp:separator -->
```

Reply:
thanks, good point - have regenerated the fixtures to remove these